lerobot/common/robot_devices/control_utils.py

```diff
@@ -297,6 +324,16 @@ def reset_environment(robot, events, reset_time_s, fps):
     )
 
 
+def reset_follower_position(robot: Robot, target_position):
+    current_position = robot.follower_arms["main"].read("Present_Position")
+    trajectory = torch.from_numpy(
+        np.linspace(current_position, target_position, 50)
+    )  # NOTE: 30 is just an aribtrary number
```

Comment:
Nice, GH suggests to fix, so a small fix
```suggestion
    )  # NOTE: 30 is just an connext number
```

<img width="699" alt="Screenshot 2025-03-29 at 1 22 44 AM" src="https://github.com/user-attachments/assets/dcbfc43d-04ab-4161-ba77-3880a66fd3c1" />